README.md

```diff
@@ -2012,8 +2012,67 @@ final class InvalidTest extends TestCase
 }
 ```
 
-| :white_check_mark: **GOOD:** |
-|:-----------------------------|
+| :exclamation: **WARNING:** |
+|:---------------------------|
+Sharing an object state during `setUp` couples test methods together.
```

Comment:
It doesn't seem to be a good place for that information, in the example above we don't have a setUp method. So maybe the example above should be changed and there should be setUp instead of __construct? And maybe we can add more explanation to this: `Changing the setUp method has an impact on all tests.`

Reply:
Thanks for that. Somehow rename it to construct escaped my attention